databrowser/src/pages/PrivacyPage.vue

```diff
@@ -1,3 +1,14 @@
 <template>
-  <div>Privacy Page</div>
+  <AppLayout>
+    <ContentArea>Privacy Page</ContentArea>
+  </AppLayout>
 </template>
+<script>
```

Comment:
I would set the `lang="ts"` info to improve tooling support (e.g. by VSCode `volar` extension).

At the moment, `volar` shows me this message:

```
Virtual script not found, may missing <script lang="ts"> / "allowJs": true / jsconfig.json.volar
```